src/operator/tensor/elemwise_unary_op.h

```diff
@@ -348,6 +351,40 @@ class UnaryOp : public OpBase {
       LogUnimplementedOp(attrs, ctx, inputs, req, outputs);
     }
   }
+
+#if MSHADOW_USE_MKL == 1
+static inline void MKL_Log(int size, const float* pIn, float* pOut) {
```

Comment:
int: rename to MKLLog

Reply:
@eric-haibin-lin Do we have any recommended convention for function/class name definition in MXNet? Just not feel very comfortable with 4 consecutive capital letters. :(